.github/workflows/windows.yml

```diff
@@ -73,5 +73,6 @@ jobs:
           export MSYSTEM_CARCH="$ARCH_${{ matrix.bits }}"
           export MSYSTEM_CHOST="$ARCH_${{ matrix.bits }}-pc-msys"
           ./scripts/build.sh --compiler ${{ matrix.compiler }} --build-type Debug --bin-path /mingw${{ matrix.bits }}/bin
+          grep '^#define' config.h
```

Comment:
Some leftover grep, I believe?

Reply:
Yup; knocked in the latest.  Thanks for catching it.